terminusdb_client/woqlclient/woqlClient.py

```diff
@@ -1108,7 +1144,9 @@ def _generate_commit(self, msg, author=None):
         ci = {"commit_info": {"author": mes_author, "message": msg}}
         return ci
 
-    def _prepare_revision_control_args(self, rc_args=None):
+    def _prepare_revision_control_args(
+        self, rc_args: Optional[dict] = None
+    ) -> Union[bool, dict]:
```

Comment:
I think it would make sense to modify this method's signature to it returns `None` instead of `False`. This would allow stricter checks in the calling code: right now there are `if rc_args:` checks which fail both for `False` and `{}`. Instead we could write `if rc_args is not None:` which is clearer. This would also make `mypy` happy in the calls to `self.dispatch()` because the types for `rc_args` would match.

Reply:
I agree. It is returning False because the js client is doing so and it makes more sense in Python to be None.

Reply:
I've amended the signature of the method to return `Optional[dict]` as discussed.